examples/04_glm_first_level/plot_localizer_surface_analysis.py

```diff
@@ -59,56 +58,46 @@
 # Project the :term:`fMRI` image to the surface
 # ---------------------------------------------
 #
-# For this we need to get a :term:`mesh`
-# representing the geometry of the surface.
-# We could use an individual :term:`mesh`,
-# but we first resort to a standard :term:`mesh`,
-# the so-called fsaverage5 template from the FreeSurfer software.
-import nilearn
+# For this we need to get a mesh representing the geometry of the surface. We
+# could use an individual mesh, but we first resort to a standard mesh, the
+# so-called fsaverage5 template from the FreeSurfer software.
 
-fsaverage = nilearn.datasets.fetch_surf_fsaverage()
```

Comment:
would we then want to deprecate this function and update `experimental.surface.load_fsaverage` to include all the mesh types?

Reply:
I'm wondering whether there is still a use case for having all the paths, not the objects.

Reply:
A path-like object can still be accessed from the `FileMesh` object created by `load_fsaverage`.

```python
from nilearn.experimental import surface
fsaverage = surface.load_fsaverage()
fsaverage['pial']['left_hemisphere'].file_path
```